graphql-api/src/main/java/com/expediagroup/streamplatform/streamregistry/graphql/mutation/StreamMutation.java

```diff
@@ -25,7 +25,7 @@
 public interface StreamMutation extends GraphQLApiType {
   Stream insert(StreamKeyInput key, SpecificationInput specification, SchemaKeyInput schema);
 
-  Stream update(StreamKeyInput key, SpecificationInput specification, SchemaKeyInput schema);
```

Comment:
wondering what client applications would be impacted due to this contract change.